plugins/org.locationtech.udig.catalog/src/org/locationtech/udig/catalog/util/ASTFactory.java

```diff
@@ -314,48 +344,56 @@ public AST getRight() {
         @SuppressWarnings("unused")
         private Literal() {/* should not be used */
         }
-        public Literal( String value ) {
+
+        public Literal(String value) {
             this.value = value;
         }
 
         /**
          * TODO summary sentence for accept ...
-         * 
+         *
          * @see org.locationtech.udig.catalog.internal.CatalogImpl.AST#accept(java.lang.String)
          * @param datum
          * @return
          */
-        public boolean accept( String datum ) {
+        @Override
+        public boolean accept(String datum) {
             return value != null && datum != null
                     && datum.toUpperCase().indexOf(value.toUpperCase()) > -1;
         }
 
         /**
          * TODO summary sentence for type ...
-         * 
+         *
          * @see org.locationtech.udig.catalog.internal.CatalogImpl.AST#type()
          * @return
          */
+        @Override
         public int type() {
             return LITERAL;
         }
-        /*
+
+        /**
          * (non-Javadoc)
-         * 
+         *
          * @see org.locationtech.udig.catalog.util.AST#getLeft()
          */
+        @Override
         public AST getLeft() {
             return null;
         }
-        /*
+
+        /**
          * (non-Javadoc)
```

Comment:
IMHO we can remove "(non-Javadoc)" and "TODO summary sentence .." comments completely where `@Override` were added. Documentation should be in interfaces ..

@sschulz92 Are you kind to remove these with this pull?